pkgs/stdenv/generic/default.nix

```diff
@@ -223,7 +223,7 @@ let
         # include it in the result, it *is* available to nix-env for queries.
         meta = { }
             # If the packager hasn't specified `outputsToInstall`, choose a default,
-            # namely `p.bin or p.out or p`;
+            # namely `lib.getBin p`;
```

Comment:
Yes, the semantics is `[(getBin pkg)]`. Thinking of it again, I can't see how to get a reasonable `pkg` argument on that line to pass it, so it might actually be best to leave the PR as it is now :-)